source/content.js

```diff
@@ -299,6 +300,10 @@ function ajaxedPagesHandler() {
 		enableFeature(extendDiffExpander);
 	}
 
+	if (pageDetect.isSingleFile() || pageDetect.isPRFiles()) {
+		enableFeature(fixStickyFileHeaderTooltips);
```

Comment:
I'm not sure if this the right place to call this script, or if the script should be added to `/features`, because this isn't necessary a "feature", but rather a fix for another feature – the sticky file-header, that is. Any thoughts?

Reply:
That's right, I think _if_ this PR gets accepted, it will have to becomes its own feature: `sticky-file-headers` or even just `sticky-headers` that injects the CSS

https://github.com/sindresorhus/refined-github/blob/f8abd1e5d7c9d3b45bf3f49e58244e01b22938ca/source/content.css#L666-L668

and also fixes the issues it causes.

That, as previously mentioned, only if it's too much trouble to fix with CSS alone, which would be preferable.